src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -144,6 +149,12 @@ public function handle()
             $helper->setCloner($cloner);
         }
 
+        $rootPath = $this->applicationRootPath;
+        if (!$rootPath) {
+            $rootPath = dirname(dirname(dirname(dirname(dirname(dirname(__DIR__))))));
```

Comment:
Thats what this PR is about. Before it only worked for the projects vendor dir. With the new api you can instruct the prettypagehandler with whatever dir/path the project is using for the vendor dependencies